book/_config.yml

```diff
@@ -27,23 +44,28 @@ latex:
 
 # Configure your Binder links, such as the URL of the BinderHub.
 launch_buttons:
-  notebook_interface: jupyterlab
-  #binderhub_url: https://gke.mybinder.org
-  # NOTE: Want to use nbgitpuller with binderhub, not sure this will work currently...
-  #binderhub_url: https://aws-uswest2-binder.pangeo.io/v2/gh/pangeo-data/notebook-binder/2021.02.02?git-pull?repo=https://github.com/uwhackweek/jupyterbook-template
-  binderhub_url: https://aws-uswest2-binder.pangeo.io
-  jupyterhub_url: https://aws-uswest2.pangeo.io
-  colab_url: "" #empty string disables button
+  binderhub_url: ''
 
 # Information about where the book exists on the web
 repository:
   url: https://github.com/uwhackweek/jupyterbook-template # Online location of your book
   path_to_book: book  # Optional path to your book, relative to the repository root
   branch: main  # Which branch of the repository should be used when creating links (optional)
 
+# Bibliography
+bibtex_bibfiles:
+  - references.bib
+sphinx:
+  config:
+    bibtex_reference_style: author_year  # or label, super, \supercite
+    templates_path: _templates
```

Comment:
need to remove `templates_path` to get the build to work. this was a hack fix for snowex google analytics.